src/Helpers.py

```diff
@@ -85,4 +87,32 @@ def _is_manualobject_enabled(multiworld: MultiWorld, player: int, object: any) -
             enabled = False
             break
 
-    return enabled
\ No newline at end of file
+    return enabled
+
+def get_items_for_player(multiworld: MultiWorld, player: int) -> List[Item]:
+    """Return list of items of a player including placed items"""
+    return [i for i in multiworld.get_items() if i.player == player]

```

Comment:
Huh, I guess I never noticed, but it's weird that `get_locations` takes an optional player id but `get_items` doesn't. 😜